ui/src/components/home/HomeContainer.tsx

```diff
@@ -60,8 +63,8 @@ const HomeContainer: FC = (): ReactElement => {
           px: ['0em', '1em'],
         }}
       >
-        <Stack direction={'column'}>
-          <Stack direction={['column', 'row']} spacing={2} gap={['25px', '12px']}>
+        <Stack direction={'column'} height="100%">
```

Comment:
as discussed, let's put a max height so it doesn't look weird on longer screens like ipad portrait orientation :D

Reply:
I put it in the container above